etna/articles/models.py

```diff
@@ -343,7 +341,6 @@ class Meta:
         BasePageWithIntro.content_panels
         + HeroImageMixin.content_panels
         + [
-            FieldPanel("author"),
```

Comment:
If you pop this back in, Wagtail should automatically give you a PageChooser here to select one.